packages/mask/src/components/shared/ApplicationBoard.tsx

```diff
@@ -211,36 +212,39 @@ function RenderEntryComponent({ application }: { application: Application }) {
     const { setDialog: setSelectProviderDialog } = useRemoteControlledDialog(
         WalletMessages.events.selectProviderDialogUpdated,
     )
-    const { closeDialog: closeApplicationBoard } = useRemoteControlledDialog(
-        WalletMessages.events.ApplicationDialogUpdated,
-    )
+
+    const { setDialog: setPersonaListDialog } = useRemoteControlledDialog(PluginNextIDMessages.PersonaListDialogUpdated)
+
+    const { setDialog: setCreatePersonaConfirmDialog } = useRemoteControlledDialog(MaskMessages.events.openPageConfirm)
+
     const ApplicationEntryStatus = useContext(ApplicationEntryStatusContext)
 
     // #region entry disabled
     const disabled = useMemo(() => {
         if (!application.enabled) return true
 
         if (application.entry.nextIdRequired) {
-            return Boolean(
-                ApplicationEntryStatus.isLoading ||
-                    ApplicationEntryStatus.isNextIDVerify === undefined ||
-                    (!ApplicationEntryStatus.isSNSConnectToCurrentPersona && ApplicationEntryStatus.isPersonaConnected),
-            )
+            return Boolean(ApplicationEntryStatus.isLoading || ApplicationEntryStatus.isNextIDVerify === undefined)
         } else {
             return false
         }
     }, [application, ApplicationEntryStatus])
     // #endregion
 
     // #region entry click effect
-    const createOrConnectPersona = useCallback(() => {
-        closeApplicationBoard()
-        ApplicationEntryStatus.personaConnectAction?.()
+    const createPersona = useCallback(() => {
+        setCreatePersonaConfirmDialog({
+            open: true,
+            type: 'dashboard',
```

Comment:
In case of ambiguity with the dashboard page.

```suggestion
            type: 'applicationBoard',
```